lib/server.js

```diff
@@ -12,6 +12,7 @@ const startServer = (config, callback) => {
   if (callback == null) callback = Function.prototype;
   const serverOpts = config.server || {};
   const port = parseInt(serverOpts.port, 10);
+  const host = config.server.hostname || 'localhost';
```

Comment:
I tried to find a way to pass a cli arg but i was not successful. Im new in node-coding this project and git and all this exciting stuff anyways. But my first PullRequest at all has passed.. yay! :-) Thanks @paulmillr !